it's not possible to to build with -DENABLE_FASTCGI=0

this commit adds it unconditionally ON for Linux:
https://github.com/facebook/hhvm/commit/773a4cd620820d8424756f6f8def466d179e5f8e

--- hhvm-HHVM-3.12.2/CMake/HPHPSetup.cmake~	2016-06-28 00:54:10.000000000 +0300
+++ hhvm-HHVM-3.12.2/CMake/HPHPSetup.cmake	2016-06-28 00:55:38.257526606 +0300
@@ -19,7 +19,9 @@
 endif()
 
 if (APPLE)
-  set(ENABLE_FASTCGI 1)
+  if(NOT DEFINED ENABLE_FASTCGI)
+    set(ENABLE_FASTCGI 1)
+  endif()
   set(HHVM_ANCHOR_SYMS
     -Wl,-u,_register_fastcgi_server
     -Wl,-segaddr,__text,0
@@ -48,14 +48,20 @@
   set(HHVM_ANCHOR_SYMS
     -Wl,--whole-archive ${HHVM_WHOLE_ARCHIVE_LIBRARIES} -Wl,--no-whole-archive)
 elseif(CYGWIN)
-  set(ENABLE_FASTCGI 0)
+  if(NOT DEFINED ENABLE_FASTCGI)
+    set(ENABLE_FASTCGI 0)
+  endif()
   set(HHVM_ANCHOR_SYMS
   -Wl,--whole-archive ${HHVM_WHOLE_ARCHIVE_LIBRARIES} -Wl,--no-whole-archive)
 elseif (MSVC)
-  set(ENABLE_FASTCGI 1)
+  if(NOT DEFINED ENABLE_FASTCGI)
+    set(ENABLE_FASTCGI 1)
+  endif()
   set(HHVM_ANCHOR_SYMS ${HHVM_WHOLE_ARCHIVE_LIBRARIES})
 else()
-  set(ENABLE_FASTCGI 1)
+  if(NOT DEFINED ENABLE_FASTCGI)
+    set(ENABLE_FASTCGI 1)
+  endif()
   set(HHVM_ANCHOR_SYMS
     -Wl,--whole-archive ${HHVM_WHOLE_ARCHIVE_LIBRARIES} -Wl,--no-whole-archive)
 endif()
